software/base/src/main/java/org/apache/brooklyn/entity/software/base/VanillaWindowsProcess.java

```diff
@@ -103,7 +103,16 @@
     // TODO If automatic updates are enabled and there are updates waiting to be installed, thirty minutes may not be sufficient...
     ConfigKey<Duration> REBOOT_COMPLETED_TIMEOUT = ConfigKeys.newDurationConfigKey("reboot.completed.timeout",
             "duration to wait whilst waiting for a machine to finish rebooting, and thus to become available again", Duration.minutes(30));
-    
-    AttributeSensor<Integer> RDP_PORT = Sensors.newIntegerSensor("rdpPort");
-    AttributeSensor<Integer> WINRM_PORT = Sensors.newIntegerSensor(WinRmTool.PROP_PORT.getName());
+
+    AttributeSensor<Integer> RDP_PORT = Sensors.newIntegerSensor("rdp.port");
+    AttributeSensor<Integer> WINRM_PORT = Sensors.newIntegerSensor("winrm.port");
+
+    @Deprecated
+    AttributeSensor<Integer> RDP_PORT_CAMEL_CASE = Sensors.newIntegerSensor("rdpPort");
+
+    @Deprecated
+    AttributeSensor<Integer> WINRM_PORT_CAMEL_CASE = Sensors.newIntegerSensor("winrmPort");
+
+    @Deprecated
+    AttributeSensor<Integer> PORT = Sensors.newIntegerSensor("port");
```

Comment:
Please rename this to `WINRM_PORT_UNQUALIFIED` or some such - calling it PORT could be confusing for those using the Java api, as they won't be able to tell what it is.